### DISCUSSION TITLE
Sort packages according to count

### DISCUSSION THREAD

**Discussion by TanmayPatil105** [TanmayPatil105/procfetch#100] [Ideas]
Should we display packages according to there count?
Currently, the output is:
```
2109 dpkg; 15 snap; 6 flatpak; 3 npm; 128 brew;
```
By sorting, it would be:
```
2109 dpkg; 128 brew; 15 snap; 6 flatpak; 3 npm;
```
Just a small idea, Thought I should document it!!
@youpong and @akshaykhoje What do you think?

**Comment by akshaykhoje** [comment]
Yeah. Sounds great! Should I do it?

**Reply by TanmayPatil105** [reply]
I'll try too!

**Reply by youpong** [reply]
Sort vector<> pkgs using [std::sort](https://en.cppreference.com/w/cpp/algorithm/sort).

**Comment by youpong** [comment]
checkout #103

**Comment by TanmayPatil105** [comment]
Cool !!
I was going to work on it @youpong !
Was busy with other stuff :)
